lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -27,7 +28,7 @@ def serializable_hash(options = {})
               end
             end
           else
-            @hash[:data] = attributes_for_serializer(serializer, @options)
+            @hash[:data] = attributes_for_serializer(serializer, options)
             add_resource_relationships(@hash[:data], serializer)
```

Comment:
Note there is some confusion in the method here between the `options` to the serializable_hash and the `@options` to the adapter, which I have resolved.